LondonBritishLibrary/orient/BLorient674.xml

```diff
@@ -158,8 +158,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                         <desc type="ReceiptNote">Four Amharic receipt notes or
                                             inventories of liturgical items and books, mentioning <persName ref="PRS13702WaldaD">Blātten Getā Walda Dāwit</persName> and 
                                             <persName ref="PRS13701Hezqeyas">Blātten Getā Ḥǝzqǝyās</persName>, <persName ref="PRS13698Amsale">ʾAzzāž ʾAmsāle</persName>,
-                                            <persName ref="PRS13697Marqorewos">Qes gabaz Marqorewos</persName>, <persName ref="">ʾAdarā Giyorgis</persName>
-                                            <persName ref="">Walda Zagabrǝʾel</persName>.</desc>
+                                            <persName ref="PRS13697Marqorewos">Qes gabaz Marqorewos</persName>,
+                                            <persName ref="PRS13712AdaraG">ʾAdarā Giyorgis</persName>, and
```

Comment:
we also have PRS1403Adaru for baldaras Adaru, a contemporary of Iyasu I

Reply:
True, but I really see not enough information for an identification (are bāldarās and baǧərond positions likely to be occupied by the same person (subsequently)?). I really don't know enough about this.

Reply:
OK let us not overidentify )